sycl/doc/GetStartedWithSYCLCompiler.md

```diff
@@ -16,39 +19,129 @@ to install.
 SYCL support: follow the instructions under
 [SYCL* Compiler and Runtimes](https://github.com/intel/llvm/releases/tag/2019-07)
 
+### Get the required tools:
+
+   a. `git` - for downloading the sources
+   
+   b. `cmake` - for building the compiler and tools (Get it at: http://www.cmake.org/download)
+      
+   c. `python` - for building the compiler and running tests (Get it at: https://www.python.org/downloads/release/python-2716/ )
+
+   d. `Visual Studio 2017 or later` (Windows only)
+
+
+---
+---
+---
+
 # Build the SYCL compiler and runtime
 
-Download the LLVM* repository with SYCL support to your local machine folder
-e.g. `$HOME/sycl` (assuming environment var `$SYCL_HOME`) folder using
-following command:
+Download the LLVM repository with SYCL support and required tools to your local machine folder, e.g. $WORK_DIR. 
+For simplicity it is assumed below that environment variable WORK_DIR contains path to an existing folder.
+
+## Set some `environment variables`. For example:
+### Linux:
+```bash
+export WORK_DIR=/export/home/work_dir
+export SYCL_HOME=$WORK_DIR/sycl_workspace
+```
+### Windows:
+Open a developer command prompt using one of tho methods:
+- Click start menu and search forthe command prompt. So, for MSVC-2017 it is '`x64 Native Tools Command Prompt for VS 2017`'
+- run 'cmd' and then '`"C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Auxiliary\Build\vcvarsall.bat" x64`'
+```bash
+set WORK_DIR=%USERPROFILE%\work_dir
+set SYCL_HOME=%WORK_DIR%\sycl_workspace
+```
+---
+## Get `OpenCL-Headers`
```

Comment:
Please, clarify that this is an optional step.
CMake will automatically configure OpenCL headers and ICD loader if OpenCL_INCLUDE_DIR and OpenCL_LIBRARY variables are not set.

BTW, these part of the instructions are not related to Windows enabling, so it would be better to have it committed separately. It should be okay to have a separate commit in this PR as long as it documentation only.

Reply:
I did not know that OpenCL-Headers and ICD-Loader parts were optional. Probably because I started SYCL build on some older machine with DLLVM_TEMPORARILY_ALLOW_OLD_TOOLCHAIN=1, etc.

Thank you. I am removing these 2 big blocks now.

Reply:
Now I am not so sure that removing those two optional blocks (download OpenCL_Headers and download/build OpenCL-ICD-Loader) was good idea.
I am going to put them to this guide again.

The reason: even with this patch (that is in code-review now): https://github.com/intel/llvm/pull/528 nmake and make work perfectly without those optional steps/blocks, while 'ninja' does not. Ninja (or our build scripts) have some problems with dependencies. Ninja does not build ICD-Loader before building sycl.lib. SYCL CMakeLists.txt look good to me.